src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -867,7 +867,7 @@ private static String processPageSourceForGodMode(String content) {
                 // now (used in comments last edited date) e.g. [Thu, 07 May 2015, 07:52:13 UTC]
                 .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC", "\\${comment\\.date}")
                 // now (used in opening time/closing time Grace period)
-                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}", "\\${grace\\.period\\.date}")
+                .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2} [AP]M", "\\${grace\\.period\\.date}")
```

Comment:
That can be confused with `${today}` (which I think is not a good name as well, but let's not touch this one now). At least there should me a mention of time. Perhaps `${dateandtime.now}` or `${datetime.now}`?